lib/mrtrix3/image.py

```diff
@@ -240,7 +240,12 @@ def statistics(image_path, **kwargs): #pylint: disable=unused-variable
   if app.VERBOSITY > 1:
     app.console('Command: \'' + ' '.join(command) + '\' (piping data to local storage)')
 
-  proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=None)
+  try:
+    from subprocess import DEVNULL
+  except ImportError:
+    import os
+    DEVNULL = open(os.devnull, 'wb')
+  proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=DEVNULL)
```

Comment:
This is necessary to prevent log level 2 level messages from being printed directly to the terminal by `mrstats` if script is run at log level 3.